### ISSUE TITLE
Upgrade to Netty 4.1.0-beta4

### ISSUE THREAD

**Issue by alkemist** [ratpack/ratpack#525]
Waiting on final release of this from Netty team.

**Comment by dsyer** [comment]
While this is in progress can you share the location of a Maven repository that has the snapshots in it? (Very naughty to release 0.9.12 BTW with snapshot dependencies, and even naughtier to put it in Maven Central.)

**Comment by alkemist** [comment]
The repo is here: `http://clinker.netty.io/nexus/content/repositories/snapshots`

Yeah, it's a bad situation. The Netty guys said they were going to release b4 before we had to do our release (but to be fair, did not promise) and the holiday break reduced the time available to revert out the b4 change. We moved versions because we hit bugs with the Netty version we were using.

Not offering ↑ as an excuse, just explaining how we ended up there.

We did mention _some_ info on this in the release notes: http://www.ratpack.io/versions/0.9.12

**Comment by dsyer** [comment]
Thanks. I got the same repo co-ordinates from the separate issue I opened: #545. That works. I'm still confused about why I couldn't find the repo location by looking in `build.gradle` though.

**Comment by johnrengelman** [comment]
It's configure in the the root project build file (`ratpack.gradle`): https://github.com/ratpack/ratpack/blob/bd8c493555e700ee908f16d42d1fde398d107df7/ratpack.gradle#L101

**Comment by dsyer** [comment]
So it is. I'm not a gradler, so I stopped looking at the first repository declaration (the "buildscript" one). Thanks for the pointer.

**Comment by johnrengelman** [comment]
Sure thing. The `buildscript` section is for declaring dependencies for the build execution itself (i.e. plugins and libraries used within the `*.gradle` files).

**Comment by davidmc24** [comment]
Unfortunately, Netty 4.1.0.Beta4 still hasn't been released.  Ratpack 0.9.13 will still have a dependency on 4.1.0.Beta4-SNAPSHOT.  Hopefully it will be out in time for Ratpack 0.9.14.
